src/workspace-data/import-jobs.test.ts

```diff
@@ -86,7 +86,7 @@ describe('useImportJobs', () => {
       const listImportJobs: MockedFn<WorkspaceContract['listImportJobs']> = jest.fn();
       listImportJobs.mockResolvedValue([{ jobId: 'job-1' }, { jobId: 'job-3' }]);
       const getImportJobStatus: MockedFn<WorkspaceContract['getImportJobStatus']> = jest.fn();
-      getImportJobStatus.mockResolvedValue({ jobId: 'job-2', status: 'Error', message: 'This job failed.' });
+      getImportJobStatus.mockResolvedValue({ jobId: 'job-2', status: 'Error', message: 'This submission failed.' });
```

Comment:
This is a different type of job and should not be renamed.

Reply:
you are correct!